packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -58,32 +83,74 @@ export class VaSegmentedProgressBar {
   }
 
   render() {
-    const { current, total, label = `Step ${current} of ${total}` } = this;
+    const { current, total, label = `Step ${current} of ${total}`, uswds, labels, centeredLabels, counters, headingText } = this;
+    let labelsArray;
+    if (labels) {
+      labelsArray = labels.split(',');
+    }
     const range = Array.from({ length: total }, (_, i) => i);
     return (
       <Host>
-        <div
-          class="progress-bar-segmented"
-          role="progressbar"
-          aria-label={label}
-          aria-valuenow={current}
-          aria-valuemin="0"
-          aria-valuemax={total}
-          aria-valuetext={label}
-        >
-          {range.map(step => (
-            <div
-              key={step}
-              class={`progress-segment ${
-                current > step ? 'progress-segment-complete' : ''
-              }`}
-            />
-          ))}
-        </div>
-        <span aria-atomic="true" aria-live="polite" class="sr-only">
-          Step {current} of {total}
-        </span>
-      </Host>
+        {
+          uswds ? (
+            <div class={`usa-step-indicator ${centeredLabels ? 'usa-step-indicator--center' : ''} ${counters && counters === 'default' ? 'usa-step-indicator--counters' : counters && counters === 'small' ? 'usa-step-indicator--counters-sm' : ''}`} aria-label="progress">
```

Comment:
This is really difficult to read. Can you import and use the `classNames` package like we've done with the other v3 components?

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-modal/va-modal.tsx#L378-L395

Reply:
Nice! 🎉 

Can you remove these lines as well?

https://github.com/department-of-veterans-affairs/component-library/blob/3c49122c2a357fbd098abfbd091ef2b5a07e37ba/packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.scss#L10-L20

Reply:
It looks like the USWDS module is expecting box-sizing to be set on html and body. The styles they have for the ::before are box-sizing:inherit. If I delete that the count circles are huge. I kept the box shadow because the 4px is a rem value in the module css.

Reply:
Gotcha. So if these USWDS style overrides are needed to account for Formation setting (or not setting) styles on the html or body tags, then this should go into the formation overrides stylesheet.

Ideally what will happen is that we won't have anything in the component stylesheets that need to be cleaned up after Formation is removed. We can just remove the formation style override import. The USWDS modules will just do their work once they have the correct body and html styles applying globally.

Reply:
Styles were moved to the formation overrides file.